fixups/RegLegacyFixups/InitializeFixup.cpp

```diff
@@ -256,6 +255,63 @@ void InitializeConfiguration()
 
                             specItem.remediationRecords.push_back(recordItem);
                         }
+
+                        // Look for DeletionMarker remediation
+                        else if (type.compare(L"DeletionMarker") == 0)
+                        {
+                            Log("RegLegacyFixups:      is DeletionMarker\n");
+                            recordItem.remeditaionType = Reg_Remediation_type_DeletionMarker;
+
+                            // Look for hive - data
+                            auto hiveType = regItemObject.try_get("hive")->as_string().wstring();
+                            traceDataStream << " hive: " << hiveType << " ;";
+                            Log(L"Hive:      %Ls\n", hiveType.data());
+
+                            if (hiveType.compare(L"HKCU") == 0)
```

Comment:
what happens in hivetype is "hkcu"?  Looks like  I can't tell if .compare is case sensitive or not.

Reply:
There is a try catch block from where we are trying to attach the dll to the process(In DllMain of PsfRuntime). These are all mandatory fields, and we don't want to load the fixup if anything fails here.

Reply:
Yes, this is not case sensitive. This shall be covered in public documentation.

Reply:
Please change this to a case-insensitive compare.  If `hiveType` is `hkcu` this should still work.

Reply:
Hi Darren, currently this is same for all other configs. We can take this as a separate task to have case insensitive comparisons for such strings. For now, keeping it as is.